elements/rh-announcement/rh-announcement.ts

```diff
@@ -0,0 +1,123 @@
+import { SlotController } from '@patternfly/pfe-core/controllers/slot-controller.js';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { LitElement, html, svg, type PropertyValues } from 'lit';
```

Comment:
Is `PropertyValues` used?

Reply:
It was in a previous version but not needed now. Removed!

Reply:
👍 

If this and other conversations have been resolved, kindly mark them as resolved ("Resolve conversation") then click "Re-request review" in the upper right in the reviewer's section when you're ready for us to take another look.